build-aux/vs2019/pastel_gtest/pastel_gtest.vcxproj

```diff
@@ -176,13 +176,15 @@
     <ClCompile Include="..\..\..\src\gtest\test_str_utils.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_tautology.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_timedata.cpp" />
+    <ClCompile Include="..\..\..\src\gtest\test_transaction.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_torcontrol.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_transaction_builder.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_trimmean.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_txid.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_uint256.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_univalue.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_upgrades.cpp" />
+    <ClCompile Include="..\..\..\src\gtest\test_util.cpp" />
     <ClCompile Include="..\..\..\src\gtest\test_utils.cpp" />
```

Comment:
please rename test_utils.cpp|h to pastel_gtest_utils.cpp|h, and replace all references in gtest.
let's keep all gtest helper staff with separate names. 
having both test_util & test_utils does not look good